pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1432,7 +1432,7 @@ func updatePipelineRunStatusFromChildRefs(logger *zap.SugaredLogger, pr *v1beta1
 // getVerificationPolicies lists the verificationPolicies from given namespace
 func getVerificationPolicies(ctx context.Context, verificationPolicyLister alpha1listers.VerificationPolicyLister, namespace string) ([]*v1alpha1.VerificationPolicy, error) {
 	var verificationpolicies []*v1alpha1.VerificationPolicy
-	if config.CheckEnforceResourceVerificationMode(ctx) || config.CheckWarnResourceVerificationMode(ctx) {
+	if config.CheckDenyVerificationNoMatchPolicy(ctx) || config.CheckWarnVerificationNoMatchPolicy(ctx) {
```

Comment:
I'll preface this with I have no idea how much actual performance benefit this would have, but in having to reach into the context for each check feels like it would be worse than doing something like:

```suggestion
	if policy := config.GetVerificationNoMatchPolicy(ctx); policy == config.NoMatchPolicyDeny || policy == config.NoMatchPolicyWarn {
```

Reply:
This happens a bunch throughout the PR

Reply:
oh ok

Reply:
I have updated all of them